main/helper.py

```diff
@@ -1,4 +1,4 @@
-mport re
+import re
```

Comment:
I think the point of refactoring was to split this functionality into multiple class and corresponding helper files based on the user's task instead of renaming the file. 
We shouldn't have a huge python utils file with 20-30 functions, we need to split this up into classes as much as we can based on user task and then have separate helper files for each of those accordingly. 
We can discuss more on this